Geolocation/README.md

```diff
@@ -0,0 +1,47 @@
+# Geolocation API
+
+The Geolocation API allows the user to provide their location to web applications if they so desire. For privacy reasons, the user is asked for permission to report location information.
+
+WebExtensions that wish to use the Geolocation object must add the "geolocation" permission to their manifest. The user's operating system will prompt the user to allow location access the first time it is requested.
+
+This feature is available only in secure contexts (HTTPS), in some or all supporting browsers.
+
+The Geolocation API is accessed via a call to navigator.geolocation; this will cause the user's browser to ask them for permission to access their location data. If they accept, then the browser will use the best available functionality on the device to access this information (for example, GPS).
```

Comment:
Until what event the location will remain accesible?
Maybe would be good a little explanation.